application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/spring/BeanConfigurator.java

```diff
@@ -134,59 +134,75 @@ public DefaultRepairConfigurationProvider defaultRepairConfigurationProvider()
+                new Class<?>[] {
+                        Supplier.class
+                }, tlsSupplier);
         try
         {
             return ReflectionUtils
                     .construct(configuration.getConnectionConfig().getCql().getProviderClass(),
-                            new Class<?>[] { Config.class, Supplier.class, CertificateHandler.class,
-                                    DefaultRepairConfigurationProvider.class },
+                            new Class<?>[] {
+                                    Config.class,
+                                    Supplier.class,
+                                    CertificateHandler.class,
+                                    DefaultRepairConfigurationProvider.class
+                            },
                             configuration, securitySupplier, certificateHandler, defaultRepairConfigurationProvider);
-        } catch (ConfigurationException e)
+        }
+        catch (ConfigurationException e)
         {
-            if (!ReloadingCertificateHandler.class.equals(certificateHandler.getClass()) && e.getCause() instanceof NoSuchMethodException)
+            if (!ReloadingCertificateHandler.class.equals(certificateHandler.getClass())
+                    && e.getCause() instanceof NoSuchMethodException)
             {
-                throw new ConfigurationException("Invalid configuration, connection provider does not support configured certificate handler", e);
+                throw new ConfigurationException("Invalid configuration, connection provider does not support"
+                        + " configured certificate handler", e);
             }
         }
 
         // Check for old versions of DefaultNativeConnectionProvider
         return ReflectionUtils
                 .construct(configuration.getConnectionConfig().getCql().getProviderClass(),
-                        new Class<?>[]{ Config.class, Supplier.class, DefaultRepairConfigurationProvider.class },
+                        new Class<?>[]{
+                                Config.class, Supplier.class, DefaultRepairConfigurationProvider.class
+                        },
                         configuration, securitySupplier, defaultRepairConfigurationProvider);
     }
 
     @Bean
-    public JmxConnectionProvider jmxConnectionProvider(Config config) throws ConfigurationException
+    public JmxConnectionProvider jmxConnectionProvider(final Config config) throws ConfigurationException
     {
         return getJmxConnectionProvider(config, jmxSecurity::get);
     }
 
-    private static JmxConnectionProvider getJmxConnectionProvider(Config configuration,
-            Supplier<Security.JmxSecurity> securitySupplier) throws ConfigurationException
+    private static JmxConnectionProvider getJmxConnectionProvider(final Config configuration,
+                                                                  final Supplier<Security.JmxSecurity> securitySupplier
```

Comment:
this looks a bit strange for me.

Reply:
Having ") throws ..." on a new line. I would prefer to end with ) and have throws on new line instead.

Reply:
Fixed